web/servlet/home/Home_de.html

```diff
@@ -66,6 +66,12 @@
                                 <!-- should have open/download button as appropriate -->
                                 <a class="btn btn-default" href="http://www.withrottle.com" role="button">Erzähle weiter »</a>
                             </p>
+                            <h4><a href="https://www.microsoft.com/store/apps/9NBLGGH4VF5J">Train Throttle</a></h4>
+                            <p><em>Train Throttle</em> ist eine native Drossel für Windows 10-Geräte. Es ist optimiert für tablets und mobile.</p>
```

Comment:
I think you want "Fahrregler" (speed controller) instead of "Drossel" (see http://www.ebay.de/bhp/fahrregler?clk_rvr_id=1106362643255&rmvSB=true and http://www.ebay.de/sch/Modellbau/22128/i.html?_from=R40&_nkw=Drossel)